### ISSUE TITLE
Stopped working suddenly

### ISSUE THREAD

**Issue by BURN-MICROSUCK** [mozilla/contain-facebook#961]
This add-on just doesn't work since i think yesterday or the day before. Tabs from facebook/meta/instagram now just open in whatever container the current tab is using. Incidentally, the "Facebook" container disappeared. Uninstalling and reinstalling the add-on, as well as re-creating a "Facebook" container didn't fix the issue.

Steps to reproduce:

1. Open browser.
2. Go to any facebook / instagram page.
3. Look at the tab at top of screen.

**Comment by agusdallalba** [comment]
The extension works if I downgrade to Firefox 121.0.1. Perhaps it's a bug in the browser?

**Comment by victorz** [comment]
When I open messenger.com,

- it doesn't show me a visual indicator on the tab anymore:
  ![no visual indicator](https://github.com/mozilla/contain-facebook/assets/7494/f2faed73-67e4-4104-a193-6704052230c6)
- I can no longer explicitly/manually open tabs in the Facebook container anymore (it's nowhere to be found like you say)
- I do however see a "Facebook [fence icon]" thing inside of the URL bar: 
![image](https://github.com/mozilla/contain-facebook/assets/7494/e3a3ba24-ef0e-44a1-8f83-49387f5c16b3)

So I guess it is actually working, but the bug is that the visual appearance on the tab itself is gone for some reason? I can't tell if it's working though. Not sure how to test that.

**Comment by BURN-MICROSUCK** [comment]
> Not sure how to test that.

In theory, if you disable the extension then from another container login to a different facebook account (or log out from your other/facebook container) then close the tab(s) and re-enable the extension, after that if you go to any facebook/meta website (from any container) you should NOT be logged in to the account you used on the different container.

**Comment by kkarpieszuk** [comment]
I had the same. The Facebook context just lost its color. 

Go to Settings > find Context using top search bar > Customize
Find Facebook on the list and click on settings
Choose some color (you will see the white color is preselected, choose a different one).

**Comment by kkarpieszuk** [comment]
> no idea how you found "Context"

@victorz you are right, it is Container. I am using Polish version of Firefox where they translated "Containers" into Polish "Konteksty" which I incorrectly translated back to "Contexts"

Anyway, I updated my comment. 

And btw, while black color does not work you, for me it is the white color. I guess it depends on the system color settings (I am using Ubuntu with dark mode preferred)
